Cargo.toml

```diff
@@ -19,8 +19,8 @@ dirs = "4.0"
 dunce = "1.0"
 env_logger = "0.9"
 futures = "0.3"
-hippo-openapi = "0.5"
-hippo = { git = "https://github.com/deislabs/hippo-cli"}
+hippo-openapi = "0.8"
+hippo = { path = "../hippo-cli"}
```

Comment:
Needs to be updated when hippo-cli PR is merged https://github.com/deislabs/hippo-cli/pull/141
Left here so anyone can easily test.

Reply:
updated! thanks